sniffles/ruletrafficgenerator.py

```diff
@@ -1631,95 +1638,165 @@ def clear_globals(self):
         global MAC_IP_MAP
         global VENDOR_MAC_DIST_DOMAIN
         global VENDOR_MAC_DIST
+        global VENDOR_MAC_DIST_OPTION
+
+        global VENDOR_MAC_DIST_RESET
+
+        VENDOR_MAC_DIST_RESET = True
         MAC_IP_MAP = dict()
-        VENDOR_MAC_DIST_DOMAIN = 100
+
+        VENDOR_MAC_DIST_DOMAIN = {}
+        VENDOR_MAC_DIST_DOMAIN['src'] = 100
+        VENDOR_MAC_DIST_DOMAIN['dest'] = 100
+
         VENDOR_MAC_DIST = {}
+        VENDOR_MAC_DIST['src'] = {}
+        VENDOR_MAC_DIST['dest'] = {}
 
-    def create_vendor_mac_dist(self, dist_file):
+    def create_vendor_mac_dist(self, dist_file, src=None, dest=None):
```

Comment:
dist_file is no longer used here, right?  If so, we can remove it as a parameter.